src/adapters/flow_node_instance_http_adapter.ts

```diff
@@ -143,6 +143,17 @@ export class FlowNodeInstanceHttpAdapter extends BaseAdapter implements Adapters
     await this.httpClient.delete(url, requestAuthHeaders);
   }
 
+  public async reset(flowNodeInstanceId: string, identity?: IIdentity): Promise<void> {
+
+    const endpoint = restSettings.paths.resetFlowNodeInstance
+      .replace(restSettings.params.flowNodeInstanceId, flowNodeInstanceId);
+
+    const url = this.buildUrl(endpoint);
+    const requestAuthHeaders = this.createRequestAuthHeaders(identity);
+
+    await this.httpClient.delete(url, requestAuthHeaders);
```

Comment:
Im Http Controller ist diese Route als "put" registriert:

https://github.com/atlas-engine/RuntimeDomain.PersistenceApi/pull/3/files#diff-b506ac2a8108ba345e95eaace69fe98dR79

Das macht denke ich auch mehr Sinn, da die FlowNodeInstanz ja nicht gelöscht, sondern nur ihr State "zurückgesetzt" wird.

```suggestion
    await this.httpClient.put(url, {}, requestAuthHeaders);
```

Reply:
Jau stimmt, sollte ein `PUT` sein 👍